src/desktop/components/artwork_columns/view.coffee

```diff
@@ -74,15 +76,30 @@ module.exports = class ArtworkColumns extends Backbone.View
 
   render: =>
     # Render columns and set some styles according to view params
-    @$el
-      .html artworkColumns
-        numberOfColumns: @numberOfColumns
-        buttonLabel: @buttonLabel()
-        seeMore: @seeMore
 
-    @sizeColumns()
-
-    @appendArtworks @collection.models
+    if ENABLE_EXPERIMENTAL_STITCH_INJECTION
```

Comment:
i'm concerned we'll create a proliferation of these feature flag switches in our code - is there a reason we don't want to just QA this and roll it out?

Reply:
OK great - as discussed, we'll enable it on Staging for a period.